pyrail/models.py

```
@@ -263,14 +263,15 @@ class Alert(DataClassORJSONMixin):
     id: str  # Alert ID
     header: str  # Alert header
     lead: str  # Alert lead
-    link: str  # Link to more information
     start_time: datetime = field(
         metadata=field_options(alias="startTime", deserialize=lambda x: timestamp_to_datetime(x))
     )  # Start time of the alert
     end_time: datetime = field(
         metadata=field_options(alias="endTime", deserialize=lambda x: timestamp_to_datetime(x))
     )  # End time of the alert
-
+    link: Optional[str] = field(
```

Ruff (0.8.2)

Undefined name `Optional`

(F821)